### PR TITLE
fix(alias): allow clearing alias from chatformheader

### PR DESCRIPTION
**Pull request by anthonybilinski**
Now has the same logic as FriendWidget. Before clearing the field would result in no change being made, instead of clearing the alias.

- [ ] Commits follow our [git commit guidelines](https://github.com/qTox/qTox/blob/master/CONTRIBUTING.md#git-commit-guidelines)

---
This change is [Reviewable](https://reviewable.io/reviews/qtox/qtox/5608)

### CODE REVIEW

File: src/widget/chatformheader.cpp
Comment (Diadlo) [inline src/widget/chatformheader.cpp]: AFAIK you can connect signal to signal. Just use
`connect(nameLabel, &CroppingLabel::editFinished, this, &ChatFormHeader::nameChanged);`

**Review by anthonybilinski — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/qtox/qtox/5608)* status: 0 of 1 LGTMs obtained

___
*[src/widget/chatformheader.cpp, line 124 at r1](https://reviewable.io/reviews/qtox/qtox/5608#-LbwNfGh4e063-bHBC9X-r1-124:-LbwkZco1j94VYMpERyx:b-4pbwau) ([raw file](https://github.com/qtox/qtox/blob/5dab231f91cc41c5a4507b451b0c458c89c8feda/src/widget/chatformheader.cpp#L124)):*
<details><summary><i>Previously, Diadlo (Polshakov Dmitry) wrote…</i></summary><blockquote>

AFAIK you can connect signal to signal. Just use
`connect(nameLabel, &CroppingLabel::editFinished, this, &ChatFormHeader::nameChanged);`
</blockquote></details>

Ooops good point. Done.

**Review by Diadlo — APPROVED** [review]

**Comment by anthonybilinski** [comment]
Is this missing an LGTM, or intentionally not passing PR checks and waiting for a second review?

**Review by Diadlo — APPROVED** [review]
:lgtm_strong:

> Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/qtox/qtox/5608#-:-Lc-J6av8U6ykZlgMhyY:b-jbsf8z)* status: :shipit: complete! 1 of 1 LGTMs obtained

**Comment by Diadlo** [comment]
@anthonybilinski It's because LGTM was not from Reviewable

**Pull request merged**
